examples/rmg/commented/input.py

```diff
@@ -204,6 +204,9 @@
     keepIrreversible=False,
     # Allows families with three products to react in the diverse direction (default).
     trimolecularProductReversible=True,
+    # Allows a seed to be saved every n iterations.
+    # The default of -1 causes the iteration to only be saved at the end of the RMG job
+    saveSeedModulus=-1
```

Comment:
Is this `saveSeedModulus` in the documentation?

Reply:
You are right, we forgot to add this to the main documentation. It made it into the commented input file, but it needs to be added to `RMG-Py/users/rmg/input.rst` as well.

Reply:
Thanks for catching this. I working on an explanation of how to use this new feature and will submit the PR today :)